### DISCUSSION TITLE
Color and typography variations/options in WP 6.6

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#275] [Topic Ideas]
WordPress 6.6 will ship with a new global color and typography variations system. Similar to global style variations, which are placed under a theme's `/styles` folder, these new variations will work on top of the same underlying system. 

However, color and typography variations let themers define specific color schemes and typography sets that users can choose from:

![image](https://github.com/WordPress/developer-blog-content/assets/1816309/daefc48a-c186-49ea-a152-748f4985dcaf)

This recreates a feature that is popular in classic themes that utilize the customizer to give the user pre-built color schemes and typography sets to choose from. Essentially, it lets the user quickly pick their colors and typography at the click of a button without affecting other parts of the overall theme design.

These new "preset" variations can also be used alongside normal global style variations, mixing and matching different designs.

Feature addition:

- https://github.com/WordPress/gutenberg/issues/61215

**Comment by bph** [comment]
The topic is approved, please continue the discussion on the issue

- #282

**Comment by bph** [comment]
Published: [Mixing and matching styles, colors, and typography in WordPress 6.6](https://developer.wordpress.org/news/2024/07/22/mixing-and-matching-styles-colors-and-typography-in-wordpress-6-6/)
